internal/output/json.go

```diff
@@ -119,6 +114,16 @@ func prepareJSONValue(v interface{}) interface{} {
 	return v
 }
 
+// StructuredError communicates that an error happened due to output that was meant to be structured but wasn't.
+type StructuredError struct {
+	Message string   `json:"error"`
+	Tips    []string `json:"tips,omitempty"`
+}
+
+func (s StructuredError) EError() string {
```

Comment:
Is this name intentional? Or was it a typo?

Reply:
Woops, typo.